patches/net/minecraft/world/level/storage/LevelStorageSource.java.patch

```diff
@@ -1,5 +1,14 @@
 --- a/net/minecraft/world/level/storage/LevelStorageSource.java
 +++ b/net/minecraft/world/level/storage/LevelStorageSource.java
+@@ -287,7 +_,7 @@
+             WorldDataConfiguration worlddataconfiguration = readDataConfig(p_307300_);
+             LevelSettings levelsettings = LevelSettings.parse(p_307300_, worlddataconfiguration);
+             FeatureFlagSet featureflagset = parseFeatureFlagsFromSummary(p_307300_);
+-            boolean flag1 = FeatureFlags.isExperimental(featureflagset);
++            boolean flag1 = FeatureFlags.isExperimental(featureflagset) || !worlddataconfiguration.enabledModdedFeatures().isEmpty();
```

Comment:
Could put this on a new line to avoid the removed line in the patch, but I'm not super picky here